### PR TITLE
[FtpUploadV1] Migrated legacy tests

### PR DESCRIPTION
**Pull request by max-zaytsev**
**Task name**: FtpUploadV1

**Description**: Migrated legacy tests

**Documentation changes required:** No

**Added unit tests:** Yes

**Attached related issue:** [#353](https://github.com/microsoft/build-task-team/issues/353)

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/FtpUploadV1/Tests/L0.ts [hunk 1]
```diff
@@ -1,16 +1,149 @@
...
+        tr.run();
+
+        assert(tr.stdOutContained('Input required: serverEndpoint'), 'Should have printed: Input required: serverEndpoint');
+        assert(tr.failed, 'task should have failed');
+
+        done();
+    });
+
+    it('check args: no rootFolder', (done: Mocha.Done) => {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT || '') || 20000);
```
Comment (anatolybolshakov) [inline Tasks/FtpUploadV1/Tests/L0.ts:24]: Do we need to set it for every test or is it enough to set it in beginning of describe?

**Comment by anatolybolshakov** [comment]
Thanks for migrating this! Do we need to remove legacy tests also?
I would also suggest to add some tests for successful - but maybe it makes sense to just create separate ticket for it.

**Review by egor-bryzgalov — APPROVED** [review]

File: Tasks/FtpUploadV1/task.json [hunk 2]
```diff
@@ -18,8 +18,8 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 165,
```
Comment (anatolybolshakov) [inline Tasks/FtpUploadV1/task.json:21]: Looks like there are changes not related to tests migration

Comment (max-zaytsev) [reply Tasks/FtpUploadV1/task.json]: Thank you! Resolved by merging task migration PR

File: Tasks/FtpUploadV1/Tests/L0.ts (hunk 1 above)
Comment (max-zaytsev) [reply Tasks/FtpUploadV1/Tests/L0.ts]: Removed timeout from tests

**Comment by max-zaytsev** [comment]
@anatolybolshakov 

>Thanks for migrating this! Do we need to remove legacy tests also?

Removed

>I would also suggest to add some tests for successful - but maybe it makes sense to just create separate ticket for it.

Created a ticket: [#630](https://github.com/microsoft/build-task-team/issues/630)

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**